src/amo/components/OverallRating.js

```diff
@@ -17,29 +38,58 @@ export class OverallRatingBase extends React.Component {
       i18n.gettext('How are you enjoying your experience with %(addonName)s?'),
       { addonName });
 
+    // TODO: Disable rating ability when not logged in
+    // (when state.auth is empty)
+
     return (
       <div className="OverallRating">
-        <p>{prompt}</p>
         <form action="">
-          <input id="OverallRating-love-it" value="love-it" name="overall-rating" type="radio" />
-          <label htmlFor="OverallRating-love-it" className="OverallRating-love-it">
-            {/* L10n: This should be a very short phrase for layout reasons, if possible. */}
-            {i18n.gettext('Love it!')}
-          </label>
-          <input id="OverallRating-it-is-ok" value="it-is-ok" name="overall-rating" type="radio" />
-          <label htmlFor="OverallRating-it-is-ok" className="OverallRating-it-is-ok">
-            {/* L10n: This should be a very short phrase for layout reasons, if possible. */}
-            {i18n.gettext("It's OK")}
-          </label>
-          <input id="OverallRating-huh" value="huh" name="overall-rating" type="radio" />
-          <label htmlFor="OverallRating-huh" className="OverallRating-huh">
-            {/* L10n: This should be a very short phrase for layout reasons, if possible. */}
-            {i18n.gettext('Huh?')}
-          </label>
+          <fieldset>
+            <legend>{prompt}</legend>
+            <div className="OverallRating-selectors">
+              <label htmlFor="OverallRating-love-it">
```

Comment:
Buttons can be associated with a label but since you have text inside the button the label seems redundant.